# codon_papers
This folder contains papers that are relevant for codon usage analyses. This is by NO means a comprehensive list and I will try to update it as I find or remember new papers. It is split into the following folders with the papers


## Foundational Papers

1. Ikemura, Toshimichi. "Correlation between the abundance of yeast transfer RNAs and the occurrence of the respective codons in protein genes: differences in synonymous codon choice patterns of yeast and Escherichia coli with reference to the abundance of isoaccepting transfer RNAs." Journal of molecular biology 158.4 (1982): 573-597.
2. Wright F. The ‘effective number of codons’ used in a gene. Gene. 1990 Mar 1;87(1):23-9.
3. Bulmer M. The selection-mutation-drift theory of synonymous codon usage. Genetics. 1991 Nov 1;129(3):897-907.
4. Akashi H. Synonymous codon usage in Drosophila melanogaster: natural selection and translational accuracy. Genetics. 1994 Mar 1;136(3):927-35.
5. Sharp PM, Stenico M, Peden JF, Lloyd AT. Codon usage: mutational bias, translational selection, or both?. Biochemical Society Transactions. 1993 Nov;21(4):835-41.
6. Roller M, Lucić V, Nagy I, Perica T, Vlahoviček K. Environmental shaping of codon usage and functional adaptation across microbial communities. Nucleic acids research. 2013 Oct 1;41(19):8842-52.
7. Trotta E. Selection on codon bias in yeast: a transcriptional hypothesis. Nucleic acids research. 2013 Nov 1;41(20):9382-95.
8. Zhou M, Guo J, Cha J, Chae M, Chen S, Barral JM, Sachs MS, Liu Y. Non-optimal codon usage affects expression, structure and function of clock protein FRQ. Nature. 2013 Mar;495(7439):111-5.
9. Kessler MD, Dean MD. Effective population size does not predict codon usage bias in mammals. Ecology and evolution. 2014 Oct;4(20):3887-900.
10. Presnyak V, Alhusaini N, Chen YH, Martin S, Morris N, Kline N, Olson S, Weinberg D, Baker KE, Graveley BR, Coller J. Codon optimality is a major determinant of mRNA stability. Cell. 2015 Mar 12;160(6):1111-24.
11. Zhou Z, Dang Y, Zhou M, Li L, Yu CH, Fu J, Chen S, Liu Y. Codon usage is an important determinant of gene expression levels largely through its effects on transcription. Proceedings of the National Academy of Sciences. 2016 Oct 11;113(41):E6117-25.  
12. Li C, Qian W, Maclean CJ, Zhang J. The fitness landscape of a tRNA gene. Science. 2016 May 13;352(6287):837-40.
13. Chen S, Li K, Cao W, Wang J, Zhao T, Huan Q, Yang YF, Wu S, Qian W. Codon-resolution analysis reveals a direct and context-dependent impact of individual synonymous mutations on mRNA level. Molecular Biology and Evolution. 2017 Nov 1;34(11):2944-58.
14. Frumkin I, Lajoie MJ, Gregg CJ, Hornung G, Church GM, Pilpel Y. Codon usage of highly expressed genes affects proteome-wide translation efficiency. Proceedings of the National Academy of Sciences. 2018 May 22;115(21):E4940-9.
15. Yannai A, Katz S, Hershberg R. The codon usage of lowly expressed genes is subject to natural selection. Genome biology and evolution. 2018 May;10(5):1237-46.
16. Lebeuf-Taylor E, McCloskey N, Bailey SF, Hinz A, Kassen R. The distribution of fitness effects among synonymous mutations in a gene under directional selection. Elife. 2019 Jul 19;8:e45952.


## Reviews of Codon Usage

1. Hershberg R, Petrov DA. Selection on codon bias. Annual review of genetics. 2008 Dec 1;42:287-99.
2. Plotkin JB, Kudla G. Synonymous but not the same: the causes and consequences of codon bias. Nature Reviews Genetics. 2011 Jan;12(1):32-42.
